lib/current_player.rb

```diff
@@ -0,0 +1,13 @@
+def turn_count(board)
+  counter = 0
+  board.each do |token|
+    if token == "X" || token == "O"
+      counter += 1
+    end
+  end
+  counter
```

Comment:
That's a great implementation.

Watch this though:

```ruby
def turn_counter(board)
 board.count{|t| t == "X" || t == "O"}
end
```

[#count](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-count)